_layouts/documentation.html

```diff
@@ -27,7 +27,7 @@
           <div class="usa-display">{{ page.title }}</div>
 
           {% if page.lead %}
-            <p class="usa-intro">{{ page.lead }}</p>
+            <p class="usa-intro">{{ page.lead | markdownify }}</p>
```

Comment:
Hm, unfortunately it looks like `markdownify` will add its own wrapping `<p>` tag which is breaking the formatting from `usa-intro`. Maybe we could swap the wrapper with a `<div>` ?

```suggestion
            <div class="usa-intro">{{ page.lead | markdownify }}</div>
```